e2e-tests/portal/cypress/integration/journeys/number-generator/test-data/baseBssDeal.js

```diff
@@ -0,0 +1,196 @@
+exports.BASE_DEAL = {
+  submissionType: 'Automatic Inclusion Notice',
+  bankInternalRefName: 'abc-1-def',
+  additionalRefName: 'Tibettan submarine acquisition scheme',
+  status: "Ready for Checker's approval",
+  previousStatus: 'Draft',
+  bank: {
+    id: '9',
+    name: 'UKEF test bank (Delegated)',
+    emails: ['maker@ukexportfinance.gov.uk', 'checker@ukexportfinance.gov.uk'],
+  },
+  maker: {
+    _id: '5f3ab3f705e6630007dcfb25',
+    username: 'maker1@ukexportfinance.gov.uk',
+    roles: ['maker'],
+    bank: {
+      id: '9',
+      name: 'UKEF test bank (Delegated)',
+      emails: ['maker@ukexportfinance.gov.uk', 'checker@ukexportfinance.gov.uk'],
+    },
+    lastLogin: '1610710442797',
+    firstname: 'Hugo',
+    surname: 'Drax',
+    email: 'maker1@ukexportfinance.gov.uk',
+    timezone: 'Europe/London',
+    'user-status': 'active',
+  },
+  details: {},
+  submissionDetails: {
+    'indemnifier-address-country': {
+      code: 'GBR',
+      name: 'United Kingdom',
+    },
+    'indemnifier-address-line-3': '',
+    'indemnifier-address-line-1': '27a',
+    'indemnifier-address-line-2': 'Maxwell Road',
+    'indemnifier-address-postcode': 'HA6 2XY',
```

Comment:
Anonymise the post code please.

Reply:
Done - there's some other files that don't have anonymised postcode/towns - can update these in another PR.